assets/src/components/detours/diversionPage.tsx

```diff
@@ -39,7 +39,7 @@ export const DiversionPage = ({
       <div className="l-diversion-page__panel bg-light">
         <DiversionPanel
           directions={directions}
-          missedStops={missedStops}
+          missedStops={missedStops || undefined}
```

Comment:
This is sort of a philosophical thing, but `DiversionPage` doesn't feel like the right place to put this. As far as `missedStops` is concerned, `DiversionPage` is just a wire that transmits data from `useDetour` to `DiversionPanel`, so I'd rather have the logic for translating _generic-falsy-value_ into `undefined` live in `useDetour` or `DiversionPanel`.

Reply:
How would you feel about just changing the type of `missedStops` returned from `useDetour()` from `Stop[] | null` to `Stop[] | undefined` much like what `useDetourDirections()` does?

Reply:
I'd be fine with that

Reply:
cf06a98a